src/main/java/org/cactoos/map/Synced.java

```diff
@@ -70,9 +70,12 @@ public Synced(final Map<X, Y> map, final Map.Entry<X, Y>... list) {
      * @param <Z> Type of items in the list
      * @checkstyle ParameterNumberCheck (5 lines)
      */
-    public <Z> Synced(final Func<Z, X> key,
-        final Func<Z, Y> value, final Map<X, Y> map,
-        final Iterable<Z> list) {
+    public <Z> Synced(
+        final Func<Z, X> key,
+        final Func<Z, Y> value,
```

Comment:
@andreoss here too let's apply the same concepts

Reply:
@victornoel Done